clusterloader2/pkg/measurement/common/probes/probes.go

```diff
@@ -228,46 +227,23 @@ func (p *probesMeasurement) checkProbesReady() (bool, error) {
 		p.framework.GetClientSets().GetClient(), isProbeTarget, expectedTargets)
 }
 
-func isProbeTarget(t prometheus.Target) bool {
-	return t.Labels["namespace"] == probesNamespace
-}
-
-func parseThresholds(params map[string]interface{}) (map[string]*measurementutil.LatencyMetric, error) {
-	thresholds := make(map[string]*measurementutil.LatencyMetric)
-	t, ok := params["thresholds"]
-	if !ok {
-		return thresholds, nil
-	}
-	for name, thresholdVal := range t.(map[string]interface{}) {
-		threshold, err := time.ParseDuration(thresholdVal.(string))
-		if err != nil {
-			return nil, err
-		}
-		thresholds[name] = makeLatencyThreshold(threshold)
+func (p *probesMeasurement) createSummary(latency measurementutil.LatencyMetric) (measurement.Summary, error) {
+	content, err := util.PrettyPrintJSON(&measurementutil.PerfData{
+		Version: currentProbesMetricsVersion,
+		// azdas
+		DataItems: []measurementutil.DataItem{latency.ToPerfData("Probes")},
```

Comment:
I guess this needs to be specific probes name now?

Reply:
Unfortunately to keep existing summaries compatible with this change we need to use hardcoded "Probes" here.

Reply:
Added comment in the code.